cmd/cadvisor.go

```diff
@@ -170,7 +173,12 @@ func main() {
 		klog.Fatalf("Failed to create a manager: %s", err)
 	}
 
-	mux := http.NewServeMux()
+	// setup open telemetry
+	if *traceEndpoint != "" {
+		defer opentelemetry.InitTrace(*traceEndpoint, *collectorCert).Stop()
```

Comment:
I think I understand why `Stop()` call is deferred but why is `InitTrace()` call deferred?